### PR TITLE
Rs 2022 08 09

### PR DESCRIPTION
**Pull request by rschlaikjer**
Me throwing up my hands at this goddamn file
2022-08-11-225744_551x583_scrot

### CODE REVIEW

File: script/Ciel/Day 5/05_01_CIEL05_3E.txt [hunk 1]
```diff
@@ -251,82 +262,91 @@ Inside is―――
...
 -- 氷河によって波だつ足場。
-Footsteps wobbled by icy torrents.
+They struggle to keep their footing atop the frozen glacier.
 }
 [sha:281454b173d78fb7f73da5693ad9f6352798dbff]{
 -- Page 595, Offset 12414.
 -- 呼吸さえ死に繋がる極限状態で、氷弾と弾丸がぶつかり合う。
-Ice projectiles and bullets are colliding in this extreme situation where even breathing can lead to death,
+Under these extreme conditions,
+ where even breathing in could spell death,
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_3E.txt:319]: "breathing in" doesn't really parse right in this sentence. just "breathing" would be my suggestion

File: script/Ciel/Day 5/05_01_CIEL05_3E.txt [hunk 2]
```diff
@@ -251,82 +262,91 @@ Inside is―――
...
 -- その戦闘時間自体が、既に一つの奇蹟だった。
-The duration of this fight itself is already a miracle.
+That it's lasted this long is already a miracle in itself.
 }
 [sha:1cdc7a9ca3e3d73cf76f521667c95826db9f09cd]{
 -- Page 596, Offset 12417.
 -- 7.62mmの火花を散らす投石機。
-A catapult making 7.62mm sparks fly.
+// Faux-metric value spotted
+// Localized to America units because otherwise 7.62mm < 8 < 9mm and that
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_3E.txt:336]: Are you sure this is a real concern? Mind you I have not only americabrain but rural americabrain so I'm not a great source here, but 7.62 I'd imagine is pretty well known to people who play video games thanks to AK-47 ubiquity 

It doesn't really matter either way

File: script/Ciel/Day 5/05_01_CIEL05_3E.txt [hunk 3]
```diff
@@ -508,27 +537,30 @@ A cursed iron dress.
...
 -- Page 610, Offset 12456.
 -- だが。その特性を、彼女はむしろ良しと利用した。
-However, she instead uses that characteristic to her own advantage.
+However―――she instead uses that characteristic to her own advantage.
 }
 [sha:e7489ea2620b946674a395742ad390b645a062e9]{
 -- Page 611, Offset 12457.
 -- “使用者が死ぬまで<外|はず>す事はできない”
-'The wearer cannot take it off until death.'
+'The wearer cannot break free of it until they are already dead.'
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_3E.txt:550]: maybe drop "of it"

File: script/Ciel/Day 5/05_01_CIEL05_3E.txt [hunk 4]
```diff
@@ -740,43 +787,55 @@ As far as the executor is concerned, her true weapon of choice is the Mystic Cod
...
 -- Page 627, Offset 12501.
 -- 少女は自らに向けて詠唱する。
-The girl chants unto herself.
+The girl chants to herself.
 }
 [sha:bdcd145c5f7f1263009ebe4b74c81ce2549b2744]{
 -- Page 627, Offset 12502.
 -- 現状、銃器による制圧は不可能だと判断した。
-In her current state, she has come to the conclusion that she cannot overwhelm him with her firearm.
+Having evaluated the situation, she's come to the conclusion that she cannot overwhelm him with firearms.
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_3E.txt:795]: with a firearm

File: script/Ciel/Day 5/05_01_CIEL05_3E.txt [hunk 5]
```diff
@@ -740,43 +787,55 @@ As far as the executor is concerned, her true weapon of choice is the Mystic Cod
...
+ <Death by Concussion|B r e a k           >.
+ <Death by Dementia|L o s t        >.
+ <Death by Torture|P a i n          >.
+ <Death by Condemnation|P u n i s h          >.
 }
 [sha:ee574b4829e96c569175612ab0cad610df7850a6]{
 -- Page 628, Offset 12507.
 -- 曰く、法が人に割り振った七つの死因。
-As told, the Seven Causes of Death are assigned to man by the law.
+// What law? Lore point?
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_3E.txt:831]: no idea and this is not an easy thing to grep for. Ask Petrikow I guess but my first association was the [7 plagues in the bible](https://en.wikipedia.org/wiki/Plagues_of_Egypt) (though notably they don't quite match), so maybe "the laws of god"?

File: script/Ciel/Day 5/05_01_CIEL05_3E.txt [hunk 6]
```diff
@@ -791,42 +850,51 @@ With a bestial howl, the girl runs through the white hell.
...
 [sha:df9a4646d05b6a4a4c74323021ac27f3e547fe88]{
 -- Page 630, Offset 12516.
 -- ギロチンの刃を重ねた<百足|むかで>の如き<蛇腹|じゃばら><剣|けん>。
-A serpentine sword shaped like the trunk of a centipede, as though guillotine blades were stacked on top of each other.
+// 蛇腹剣 seems to be called a 'whip sword' in EN, a 'serpentine sword' seems to
+// conjure images of wavy blades instead of the telescoping / whipping thing.
+// However, later on in D5 there's some 文字通り business on 蛇, so I've kept
+// serpentine for 蛇腹 but in this instance called it a whip sword to establish
+// the connection in EN.
+A serpentine whip sword, like a centipede composed of stacked guillotine blades.
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_3E.txt:885]: whip blade maybe? "whip sword" sounds a little ridiculous

File: script/Ciel/Day 5/05_01_CIEL05_3E.txt [hunk 7]
```diff
@@ -927,37 +1002,50 @@ One of the Twenty-Seven Ancestors, Sword Monk Be'ze.
...
-On the other, the composite-convertible serpentine sword over two meters long.
+// What the FUCK is a 複合可変
+// '断面はU型と箱型の結合の複合可変断面形式で'
+// https://www.honda.co.jp/factbook/auto/LEGEND/19870206/le87-015.html
+// In this context, some sort of hybrid between u-channel and box beam?
+// '低速域から高速域まで、力強いトルクを発揮する、複合可変吸気システム'
+// https://jglobal.jst.go.jp/detail?JGLOBAL_ID=202002225713263263
+// Second one seems to be describing something like VLIM, but with _multiple_ moving pieces?
+// https://en.wikipedia.org/wiki/Variable-length_intake_manifold
+// I guess interpreting it as something that is a fusion of styles / techniques
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_3E.txt:1035]: yeah, I think you're quite overthinking it but coming to the right conclusion. Just look at the compounds 複合 and 可変: "composite" and "can be changed".

**Review by Avuxo — APPROVED** [review]
See, the problem with tsukihime remake is that there just isn't enough narration

...!

File: script/Ciel/Day 5/05_01_CIEL05_3E.txt [hunk 8]
```diff
@@ -26,97 +27,101 @@ I dash down the stairs, relying on the emergency lights.
...
 -- Page 580, Offset 12366.
 -- だから―――あいつに人間の常識なんてものを期待した、俺の方が馬鹿だった。
-That's why―――I was the fool for expecting her to have a human's common sense.
+%{i}I'm%{/i} the fool here―――for expecting her to act the same way a human would.
 }
 [sha:c8023926bf2476688982734cb90f0c3825a66213]{
 -- Page 581, Offset 12367.
 -- 「それでも―――期待したくて仕方がなかったんだよ、くそ！」
-"Still―――I wanted to hope she did, shit!"
+"Still―――can you fucking blame me for hoping?"
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Ciel/Day 5/05_01_CIEL05_3E.txt:73]: Hmm, we don't really have him use that contraction much elsewhere. Sounds more like a Marioism. I was on the fence about this line - it felt a bit akward to just have the cuss at the end, but `fuck` is the only word that really seems to fit mid-sentence and feels to me like it's bordering on too strong

File: script/Ciel/Day 5/05_01_CIEL05_3E.txt (hunk 2 above)
Comment (rschlaikjer) [reply script/Ciel/Day 5/05_01_CIEL05_3E.txt:336]: Well, _I_ was confused and had to go look up how bullet numbering works 
AokoHeh

I feel like 'cal' feels weightier also? I'd probably lean towards keeping it.

File: script/Ciel/Day 5/05_01_CIEL05_3E.txt (hunk 6 above)
Comment (rschlaikjer) [reply script/Ciel/Day 5/05_01_CIEL05_3E.txt:885]: I agree that it's dumb, but I think whip sword kinda encodes it better.

File: script/Ciel/Day 5/05_01_CIEL05_3E.txt (hunk 7 above)
Comment (rschlaikjer) [reply script/Ciel/Day 5/05_01_CIEL05_3E.txt:1035]: Yeah, I went on this tangent before realizing several files later that it did the extendo business, which made it a _lot_ clearer what was going on

File: script/Ciel/Day 5/05_01_CIEL05_3E.txt (hunk 2 above)
Comment (Avuxo) [reply script/Ciel/Day 5/05_01_CIEL05_3E.txt:336]: sure, not a big deal.

File: script/Ciel/Day 5/05_01_CIEL05_3E.txt (hunk 7 above)
Comment (Avuxo) [reply script/Ciel/Day 5/05_01_CIEL05_3E.txt:1035]: sure

File: script/Ciel/Day 5/05_01_CIEL05_3E.txt (hunk 8 above)
[the remark this answers is not in the document]
Comment (Avuxo) [reply script/Ciel/Day 5/05_01_CIEL05_3E.txt:73]: alright

**Pull request merged**